packages/components/src/Brand/docs/Brand.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react"
```

Comment:
Almost missed this. For the AIO, we want the docs to go into `_docs/` instead of `docs/` :)

Reply:
Yep, missed that step